sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -220,6 +220,27 @@ def display_timezone(self, value):
     """
     self._display_timezone = value
 
+  @property
+  def cache_root(self):
+    """The cache directory specified by the user.
+
+    Defaults to None.
+    """
+    return self._cache_root
+
+  @cache_root.setter
+  def cache_root(self, value):
+    """Sets the cache directory.
+
+    Defaults to None.
+
+    Example of local directory usage::
+      interactive_beam.options.cache_root = "/Users/username/my/cache/dir"
+
+    Example of GCS directory usage::
```

Comment:
Let's mark this docstring as not supported yet or delete it from the module since it's not supported yet.